src/components/NavBar/OidcLoginComponent.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from 'react';
+import { TextField, Box, Button } from '@mui/material';
+import { LoginButton } from '@inrupt/solid-ui-react';
+
+// Utility Imports
+import { SOLID_IDENTITY_PROVIDER } from '../../utils';
+
+// Custom Hook Imports
+import { useRedirectUrl } from '../../hooks';
+
+const OidcLoginComponent = () => {
+  const [oidcIssuer, setOidcIssuer] = useState(SOLID_IDENTITY_PROVIDER);
+  const redirectUrl = useRedirectUrl();
+
+  return (
+    <>
+      <Box sx={{ flexGrow: 1 }} />
+      <Box sx={{ flexGrow: 1 }} />
+      <TextField
```

Comment:
TextField is off centered vertically, but that's a minor thing. Think we can update that in a future PR for styling.

Reply:
Fixed